server/kit/log.go

```diff
@@ -104,23 +105,58 @@ func LogMsg(ctx context.Context, message string) error {
 	return level.Info(Logger(ctx)).Log("message", message)
 }
 
+// Logf will format the given string with the arguments then log to the server logger
+// with the key "message" along with all the common request headers or gRPC metadata.
+// Arguments are handlered in the manner of fmt.Printf.
+// This message will have an "info" log level associated with it.
+func Logf(ctx context.Context, format string, v ...interface{}) error {
+	return level.Info(Logger(ctx)).Log("message", fmt.Sprintf(format, v...))
+}
+
 // LogDebug will log the given message to the server logger
 // with the key "message" along with all the common request headers or gRPC metadata.
 // This message will have a "debug" log level associated with it.
 func LogDebug(ctx context.Context, message string) error {
 	return level.Debug(Logger(ctx)).Log("message", message)
 }
 
+// LogDebugf will format the given string with the arguments then log to the server
+// logger with the key "message" along with all the common request headers or gRPC
+// metadata.
+// Arguments are handlered in the manner of fmt.Printf.
+// This message will have a "debug" log level associated with it.
+func LogDebugf(ctx context.Context, format string, v ...interface{}) error {
+	return level.Debug(Logger(ctx)).Log("message", fmt.Sprintf(format, v...))
```

Comment:
Why not `return LogDebug(ctx, fmt.Sprintf(format, v...))`?

Reply:
Not a bad idea!

Reply:
Done. along with the other two one we could do this.